TicketDesk/ngWebClient-StandAlone/src/app/app-settings.ts

```diff
@@ -1,4 +1,5 @@
 export const serviceBaseURL = 'http://localhost:50067/api';
+export const adUserURL = serviceBaseURL + '/users/';
```

Comment:
we have space here for this to be `activeDirectoryUserURL`

Reply:
And I think that would make it usefully more clear

Reply:
but I'm just commenting to have you consider the idea, not a change necessary for approval